README.md

```diff
@@ -191,4 +232,4 @@ To start contributing to this plugin, review the [Contributing to Pelican][] doc
 
 [Pelican plugins documentation]: https://docs.getpelican.com/en/latest/plugins.html
 [existing issues]: https://github.com/pelican-plugins/search/issues
-[Contributing to Pelican]: https://docs.getpelican.com/en/latest/contribute.html
+[contributing to pelican]: https://docs.getpelican.com/en/latest/contribute.html
```

Comment:
Unless there's a clear reason for it, let's revert this change.

Reply:
The reason behind this was a bug in prettier.io which is resolved in [2.8.2](https://github.com/prettier/prettier/blob/main/CHANGELOG.md#282) . Will change it back to the original one.